batch-setup/provision.sh

```diff
@@ -24,6 +26,7 @@ for repo in raw_tiles tilequeue vector-datasource tileops; do
 done
 cat > /usr/local/etc/planet-env.sh << eof
 #!/bin/bash
+export GOPATH=/usr/local
```

Comment:
This seems like it would stomp on other stuff in `/usr/local/`. Should we put it in `/usr/local/gopath` and add `/usr/local/gopath/bin` to `$PATH` in `/etc/profile`?

Reply:
On second thoughts, it looks like the changes to `docker/missing-meta-tiles-write/Makefile` mean that we don't need `$GOPATH` at all?

Reply:
> This seems like it would stomp on other stuff in /usr/local/

Is there a problem with that though?

I've been trying to move away from having any go build requirements on the instance itself. I put the binaries themselves onto s3, and we're just pulling those down on startup. I wasn't sure if there was anything else that needed it, so just setting `GOPATH` seemed like a faster way to sidestep the problem.

There's code in the python that expects `GOPATH` to be set, and from what I can tell it's just to find the commands in the first place. Should we just explicitly set that instead? ie take in the path to where those commands are via a command line arg, and not set `GOPATH` at all?

Reply:
Perhaps we can make it optional? When I've been running this locally, `$GOPATH/bin` was never in my `$PATH` (and I get kinda paranoid about not having random stuff in my `$PATH`), so `$GOPATH` seemed like a sensible way to do it. Do you think we could change the script so that it looks for the binary in `$GOPATH` if it's set, in `$PATH` if not, and errors out if it doesn't find the binary in either?

Reply:
With my pedantic hat on, searching for commands in `$GOPATH` at runtime isn't ideal. It's just meant to have meaning at compile time. `$PATH` really does seem like the right place for it. If we don't want to have these commands in `$PATH`, we could just pass in the full paths to them via config or command line, or just put all of these in a separate base path these we send in via an arg.

I took the hat off, and made the change in 18315b1 :)